e2e/app/component/workspace-card.ts

```diff
@@ -41,46 +45,58 @@ export default class WorkspaceCard extends BaseElement {
   }
 
   static async findCard(page: Page, workspaceName: string): Promise<WorkspaceCard | null> {
-    const selector = `.//*[@data-test-id="workspace-card-name" and text()="${workspaceName}"]`;
+    const selector = `.//*[@data-test-id="workspace-card-name" and normalize-space(text())="${workspaceName}"]`;
     const allCards = await this.getAllCards(page);
     for (const card of allCards) {
-      const children = await card.asElementHandle().$x(selector);
+      const handle = card.asElementHandle();
+      const children = await handle.$x(selector);
       if (children.length > 0) {
         return card; // matched workspace name, found the Workspace card.
       }
-      await card.dispose(); // not it, dispose the ElementHandle.
+      await handle.dispose(); // not it, dispose the ElementHandle.
     }
     return null; // not found
   }
 
 
-  constructor(page: Page) {
-    super(page);
+  constructor(private readonly page: Page) {
```

Comment:
why this change?

Reply:
Because class is no longer extends BaseElement.